File: gluecodium/src/test/resources/smoke/field_constructors/output/android/com/example/smoke/ImmutableStructWithClash.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class ImmutableStructWithClash {
    @NonNull
    public final String stringField;
    public final int intField;
    public final boolean boolField;
    public ImmutableStructWithClash() {
        this.stringField = "nonsense";
        this.intField = 42;
        this.boolField = true;
    }
    public ImmutableStructWithClash(final boolean boolField, final int intField, @NonNull final String stringField) {
        this.boolField = boolField;
        this.intField = intField;
        this.stringField = stringField;
    }
}

File: gluecodium/src/test/resources/smoke/field_constructors/output/android-kotlin/com/example/smoke/ImmutableStructWithClash.kt
/*

 *
 */

@file:JvmName("ImmutableStructWithClashExtensions")

package com.example.smoke


class ImmutableStructWithClash {
    @JvmField val stringField: String
    @JvmField val intField: Int
    @JvmField val boolField: Boolean





    constructor() {
        this.stringField = "nonsense"
        this.intField = 42
        this.boolField = true
    }


    constructor(boolField: Boolean, intField: Int, stringField: String) {
        this.boolField = boolField
        this.intField = intField
        this.stringField = stringField
    }





}

